### ISSUE TITLE
Questions about the choice of m

### ISSUE THREAD

**Issue by asdLeaving** [Wireless3C-SJTU/CDDM-channel-denoising-diffusion-model-for-semantic-communication#2]
In the paper, the _m_ chosen according to Eq. (21) makes \frac{1-\bar{α_m}}{\bar{α_m}} close to 2σ^2, which makes \bar{α_m} approximately equal to 1/(1+2σ^2).
However, the KL divergence in Eq. (19) approaches 0 when \bar{α_m} = 1/(1+σ^2), which mismatches with the \bar{α_m} using _m_ chosen according to Eq. (21). As a consequence, the _m_ obtained by Eq. (21) cannot guarantee the KL divergence converge to 0.
I wonder if the coefficient of noise term should be 2σ^2 rather than σ^2 in Proposition 1, Eqs. (4), (11) (12) and so on. In AWGN case for the convenience of analysis, the complex symbol to be transmitted x_c has average energy of unity, and the complex channel noise has average energy of 2σ^2. The real part of the received signal has average energy of (0.5 + σ^2), thus after normalization and reshape, the y_r ∈ R^{2k} has unity average energy, of which sqrt{1/(1+2σ^2)} belongs to the signal and sqrt{(2σ^2)/(1+2σ^2)} belongs to noise.

**Comment by MOSS-550U** [comment]
Dude, I feel you. This whole thing is so confusing. Have you found any legit explanation for these issues?

**Comment by asdLeaving** [comment]
> Dude, I feel you. This whole thing is so confusing. Have you found any legit explanation for these issues?

No. The author claimed that his derivation is correct in email reply.
